mono/utils/mono-error.c

```diff
@@ -451,6 +451,18 @@ mono_error_set_argument (MonoError *oerror, const char *argument, const char *ms
 }
 
 void
+mono_error_set_argument_null (MonoError *oerror, const char *argument, const char *msg_format, ...)
+{
+	MonoErrorInternal *error = (MonoErrorInternal*)oerror;
+	mono_error_prepare (error);
+
+	error->error_code = MONO_ERROR_ARGUMENT_NULL;
+	error->type_name = argument; /*use the first available string slot*/
```

Comment:
On my recent cleanup of mono-error I fixed that mistake. We now have a field to store those so we don't break mono_errror_get_message. See https://github.com/mono/mono/commit/4df79f93834a018962eeea4426134ef0716a835f